packages/contracts-bedrock/test/libraries/TransientContext.t.sol

```diff
@@ -0,0 +1,220 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+// Testing utilities
+import { Test } from "forge-std/Test.sol";
+
+// Target contractS
+import { TransientContext } from "src/libraries/TransientContext.sol";
+import { TransientReentrancyAware } from "src/libraries/TransientContext.sol";
+
+/// @title TransientContextTest
+/// @notice Tests for the TransientContext library.
+contract TransientContextTest is Test {
+    /// @notice Slot for call depth.
+    /// @dev    Ensure that this matches the slot in the library.
+    bytes32 internal constant CALL_DEPTH_SLOT = 0x7a74fd168763fd280eaec3bcd2fd62d0e795027adc8183a693c497a7c2b10b5c;
```

Comment:
perhaps you could do the inline hash here so we know that it matches the value hardcoded in the contract